src/core/XRPValue.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const GlobalBigNumber = require('bignumber.js');
+const BigNumber = GlobalBigNumber.another({
+  ROUNDING_MODE: GlobalBigNumber.ROUND_HALF_UP,
+  DECIMAL_PLACES: 40
+});
+
+const Value = require('./value').Value;
+
+class XRPValue extends Value {
+
+  constructor(value) {
```

Comment:
That's a good idea, but there are four tests in amount-tests.js using Amount.from_quality that violate this rule even though they are using XRP.